src/other/Table.js

```diff
@@ -557,8 +603,41 @@
         HTMLWidget.prototype.exit.apply(this, arguments);
     };
         
+    Table.prototype.setColumnWidths = function() {
+        var context = this;
+        var tableMarginHeight = this.fixedHeader() ? this.thead.node().offsetHeight : 0;
+
+        var totalWidth = 0;
+        var tds = this.table.select("tbody").selectAll("tr").selectAll("td");
+        var th = this.thead.selectAll("th").data(this.showHeader() ? this.columns() : [], function (d) { return d; });
+        th.each(function(d, i) {
+            var thwidth = this.offsetWidth;
+            var tdwidth = tds.empty() || !tds[0][i] ? 0 : tds[0][i].offsetWidth;
+            var usewidth = thwidth >= tdwidth ? thwidth : tdwidth;
+            this.style.width = usewidth + "px";
+            if (!tds.empty() &&  tds[0][i]) {
+                tds[0][i].style.width = usewidth + "px";
+            }
+            totalWidth += usewidth;
+        });
+        this.thead
+            .style("position", this.fixedHeader() ? "absolute" : "relative")
+            .style("width", totalWidth + "px")
+            .style("margin-top", "0px")
+        ;
+        this.table
+            .style("width", totalWidth + "px" )
+        ;
+        this.tableDiv
+            .style("margin-top", (context.fixedHeader() ? tableMarginHeight : 0) + "px" )
+        ;
+        this.tbody
+            .style("width", totalWidth + "px" )
+        ;
+    };
+
     Table.prototype.getBodyRow = function(i) {
-        return this.table.selectAll("tbody tr")
+        return this.table.selectAll("tbody tr.trId" + this._id)
```

Comment:
@dtsnell4 This selector will grab unexpected elements when a Table widget exists within another Table widget.